client/sites-dashboard/controller.tsx

```diff
@@ -97,6 +99,11 @@ export function sitesDashboard( context: PageJSContext, next: () => void ) {
 			/>
 		</>
 	);
+
+	if ( isEnabled( 'layout/dotcom-nav-redesign-v2' ) ) {
+		// By definition, Sites Management does not select any one specific site
+		context.store.dispatch( setAllSitesSelected() );
```

Comment:
The default behaviour of sites dashboard is to auto select a site. This dispatch just clears/resets that so that the sidebar isn't automatically collapsed.